client/client_root_validation_test.go

```diff
@@ -22,6 +22,8 @@ type SignedRSARootTemplate struct {
 	RootPem string
 }
 
+var passphraseRetriever = func(string, string, bool, int) (string, bool, error) { return "passphrase", false, nil }
```

Comment:
Also, any reason it's a var, rather than `func passphraseRetriever(string, string ....`?